### PR TITLE
Update Table 1

### PR DESCRIPTION
**Pull request by d-anadria**
Added information on FASTREAD (aka. FAST2), will add more on this and other software soon.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ properties are evaluated:
...
 | [Colandr](#colandr)            | [:link:](https://hslib.jabsom.hawaii.edu/colandr)|                                                            |:white_check_mark:[:link:](https://doi.org/10.1111/cobi.13117)         |[:link:](https://hslib.jabsom.hawaii.edu/colandr/getting_started) |  |  |
-| [EPPI-Reviewer](#eppi-reviewer)| [:link:](https://eppi.ioe.ac.uk/cms/)            |                                                            |:white_check_mark:[:link:](https://doi.org/10.1007/s10664-017-9587-0)  |[:link:](https://eppi.ioe.ac.uk/cms/Default.aspx?tabid=3822)     |  |  |
-| [FASTREAD](#fastread)          |                                                  | :white_check_mark:[:link:](https://github.com/fastread/src)|                                                                       |                                                                 |  |  |
+| [EPPI-Reviewer](#eppi-reviewer)| [:link:](https://eppi.ioe.ac.uk/cms/)            | :zap:[:link:](https://doi.org/10.1016/j.eswa.2018.11.021)  |:white_check_mark:[:link:](https://doi.org/10.1007/s10664-017-9587-0)  |[:link:](https://eppi.ioe.ac.uk/cms/Default.aspx?tabid=3822)     |  |  |
```
Comment (Rensvandeschoot) [inline README.md:26]: The hyperlink https://doi.org/10.1016/j.eswa.2018.11.021 has been added to the row for EPPI-reviewer. Could you change the hyperlink to a paper introducing EPPI-reviewer? Thanks!

Comment (d-anadria) [reply README.md]: I resolved the issue. Added some additional info to Table 1.

Comment (Rensvandeschoot) [reply README.md]: Many thanks!

**Comment by Rensvandeschoot** [comment]
Thank you for your additions!

**Pull request merged**